src/app/screens/Detail/index.tsx

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
-import { Text } from 'react-native';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import { useRoute } from '@react-navigation/native';
+import imagePlaceholder from '@assets/img_book6.png';
+import { BookProps } from '@interfaces/books';
+
+import { ADD_TO_WISHLIST, RENT, AVAILABLE } from './constants';
+import styles from './styles';
 
 function Detail() {
-  // TODO: This will change in the next PR
-  return <Text>Details</Text>;
+  const route = useRoute();
+  const { title, image, author, genre, year } = route.params as BookProps;
```

Comment:
If Detail is a screen you can get the route from the params like `function Detail({route}) {` so you dont need the hook `useRoute`

Reply:
You are right. I forgot that.